acs-core/src/main/java/com/arrow/acs/client/model/ApplicationModel.java

```diff
@@ -10,6 +10,9 @@
  *******************************************************************************/
 package com.arrow.acs.client.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
 public class ApplicationModel extends DefinitionModelAbstract<ApplicationModel> {
```

Comment:
In general, it is desired to re-generate serialVersionUID if change class fields